WCF/Shared/RequestTrackingTelemetryModule.cs

```diff
@@ -85,7 +85,12 @@ void IWcfTelemetryModule.OnEndRequest(IOperationContext operation, Message reply
                 responseCode = HttpStatusCode.InternalServerError;
             }
 
-            telemetry.Success = !isFault;
+            // if the operation code has already marked the request as failed
+            // don't overwrite the value if we think it was successful
+            if ( isFault || !telemetry.Success.HasValue )
+            {
+                telemetry.Success = !isFault;
+            }
             telemetry.ResponseCode = responseCode.ToString("d");
```

Comment:
Never thought about it.... it is basically set mostly just for HTTP-based requests, but not sure why you'd want to change it from what was really returned. I can add it if you think it would be useful.